README.md

```diff
@@ -97,6 +97,14 @@ argument.
 shopify.on('callLimits', limits => console.log(limits));
 ```
 
+When using the GraphQL API, a different property is used to track the API call
```

Comment:
We should still mention that the `autoLimit` option is ignored.

Reply:
Right, on it